eks/kms.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_kms_key" "eks_node_ebs_encryption_key" {
+  description = "EKS Node EBS volume encryption key"
+
+  policy = data.aws_iam_policy_document.ebs_decryption.json
+}
```

Comment:
To make the policies above work, should also create a named alias.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kms_alias

Reply:
Ignoring aliasing for now due to circular dependency (and lack of sleep!)